components/camel-as2/camel-as2-api/src/main/java/org/apache/camel/component/as2/api/AS2ClientConnection.java

```diff
@@ -69,8 +73,16 @@ public AS2ClientConnection(String as2Version, String userAgent, String clientFqd
                 .add(new RequestExpectContinue(true)).build();
 
         // Create Socket
-        Socket socket = new Socket(targetHost.getHostName(), targetHost.getPort());
-
+        //Socket socket = new Socket(targetHost.getHostName(), targetHost.getPort());
+	
+	if (sslContext==null) {
+		socket = new Socket(targetHost.getHostName(), targetHost.getPort());
+	} else {
+		SSLSocketFactory factory = sslContext.getSocketFactory();
+		socket = (SSLSocket) factory.createSocket(targetHost.getHostName(), targetHost.getPort());
+		//socket.setEnabledProtocols(new String[]{"TLSv1.3"});
```

Comment:
Why is it commented?

Reply:
I was having some issues with setting up KeyStore/ TrustStore and got handshake failed error. So I tried changing allowed protocol to only TLSv1.3. 
Did not need it though, so I commented it at that time and forgot to remove this line.